project/Dependencies.scala

```diff
@@ -8,7 +8,16 @@ object Dependencies extends AutoPlugin {
     object Akka {
       val version = "2.6.15"
       val akkaHttpVersion = "10.2.6"
+      val alpakkaVersion = "2.1.1"
+      val managementVersion = "1.1.1"
 
+      val kafkaStream = "com.typesafe.akka" %% "akka-stream-kafka" % alpakkaVersion
```

Comment:
This already exists as Alpakka.kafka but I think Akka.kafkaStream makes more sense so we can clean up the old one